torch/onnx/symbolic_opset9.py

```diff
@@ -1002,6 +1002,18 @@ def ne(g, self, other):
     return g.op("Equal", self, other)
 
 
+def __isnot_(g, self, other):
+    if sym_help._is_none(other):
+        if sym_help._is_none(self):
+            return g.op("Constant", value_t=torch.BoolTensor([1]))
```

Comment:
I wonder if `value_t` should be `torch.tensor(1, dtype=torch.bool)` or `torch.BoolTensor([1])` (basically rank 0 or rank 1)?

Reply:
I don't think rank matters, I think we just need to agree on convention. Symbolic for aten::is_floating_point is also returning tensor of rank1